setup.cfg

```diff
@@ -9,7 +9,7 @@ python_files = *Test.py
 python_classes = *Test
 python_functions = *_test
 timeout = 35
-addopts = --color=yes
+addopts = --color=yes --ignore=test-requirements.txt
```

Comment:
According to [this](http://stackoverflow.com/questions/11117062/how-to-tell-py-test-to-skip-certain-directories) we can use [`norecursedirs`](http://pytest.org/latest/customize.html#confval-norecursedirs)